pom.xml

```
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-issue-1847-SNAPSHOT</version>
```

Version is wrong - should be the same as what you've set in the `strongbox-parent`. :) 
```suggestion
        <version>1.0-PR-62-issues-1847-SNAPSHOT</version>
```

---

But that *is* the version set in strongbox-parent. This PR is against ˋmasterˋ. #1953 is the PR against ˋissues/1649ˋ